.ci/scripts/windows/install-tools.ps1

```diff
@@ -0,0 +1,12 @@
+# Abort with non zero exit code on errors
```

Comment:
I see potential in having this script living in the shared library. wdyt?

Reply:
That's a good point, from the implementation details there are two options, I'd say let's do it in a second iteration, WDYT?

Reply:
This is now moved to a shared library step :)